src/AnyNumberContainer.java

```diff
@@ -0,0 +1,20 @@
+/*
+* Task:
+* Create class-container for numbers;
+*/
+class AnyNumberContainer <T extends Number> {
```

Comment:
You could have reused **AnyTypeContainer** here like

`class AnyNumberContainer <T extends Number> extends AnyTypeContainer<T> {`
`  AnyNumberContainer(T data) {`
`    super(data);`
`  }`
`}`